src/style.css

```diff
@@ -1 +1,75 @@
-/* add styles here */
+/* #region body */
+
+body {
+  margin: 0;
+  background-color: #fff;
+}
+
+/* #endregion */
+
+/* #region container */
+
+/* .container {
+  box-sizing: border-box;
+  width: 140px;
+  height: 160px;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+} */
+
+/* #endregion */
+
+/* #region stars */
+
+.stars--0 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--1 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--2 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--3 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--4 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--5 {
+  display: flex;
+  flex-direction: row;
+}
+
+/* #endregion */
+
+/* #region stars_star_first_row */
+
+.stars_star1 {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  margin-right: 4px;
+}
+
+.stars_star2 {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star-active.svg);
+  margin-right: 4px;
+}
```

Comment:
```suggestion

```
The active star background should depend on the class with the modifier, so consider implementing this logic